python/ray/worker.py

```diff
@@ -623,13 +623,23 @@ def init(
 
         ray.init()
 
-    To connect to an existing Ray cluster, use this as follows (substituting
-    in the appropriate address):
+    To connect a driver to an existing Ray cluster, use this as follows
+    (substituting in the appropriate address):
 
     .. code-block:: python
 
         ray.init(address="123.45.67.89:6379")
 
+    To connect to an existing Ray cluster using Ray client, specify a protocol
+    in the address:
+
+    .. code-block:: python
+
+        ray.init(address="ray://123.45.67.89:10001")
+
+    More details on Ray client usage can be found at
```

Comment:
are we updating the actual message that is output after `ray start` too?

Reply:
Sure, is this what you have in mind:
<img width="558" alt="Screen Shot 2021-07-30 at 6 04 21 PM" src="https://user-images.githubusercontent.com/14043490/127723860-7fd11db6-1568-4ecc-906b-8ee5114f5017.png">

Reply:
Love the screenshot. Can we replace "node_ip" with "head_node_ip_address" and replace "for example to connect ..." with "for example to connect to a remote cluster from your laptop directly, use... "

Reply:
<img width="564" alt="Screen Shot 2021-08-02 at 9 06 36 AM" src="https://user-images.githubusercontent.com/14043490/127891522-59382a8e-c1df-4d4f-b1ee-e874d497a540.png">